scripts/functions/installer

```diff
@@ -1100,8 +1100,15 @@ setup_user_profile_rc()
     do
       touch "$profile_file"
       printf "%b" "
-# Add RVM to PATH for scripting. Make sure this is the last PATH variable change.
+###############################################################################
+# At $(date --iso-8601=s), the RVM ${rvm_version} created
```

Comment:
I'm afraid I'm not familiar with these steps that would seem necessary for fully testing:
- Uninstalling RVM (probably easy, but maybe time-consuming)
- Installing RVM from the branch this PR is based on (trickier)

I'm afraid I can't really put any more time into the matter at the moment either, sorry :( I hope that what I've contributed so far is helpful and can make it into RVM once it has been tested/approved/tweaked by you and @mpapis .

Thanks for maintaining RVM and for your great feedback on this PR :)